Gemfile

```diff
@@ -0,0 +1,46 @@
+source 'https://rubygems.org'
+
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '4.2.10'
```

Comment:
Why are you using this version of rails?

Reply:
Can we use rails 5.0.7.2 because that is maximum version we can reachfor upgrade And I do not understand why but every time when I have tried to upgrade to version 5.1.0 I hade some dependecies issues Which i could not to resolve...